updated: update the copyright removing Muut from the distribution files

GianlucaGuarini opened the pull request:
see https://github.com/riot/riot/issues/1545

GianlucaGuarini commented:
@tipiirai could you please have a look at this pull request?

cc @aMarCruz @cognitom @rsbondi

aMarCruz commented:
Why not in alphabetical order? he...
I think it's better set the year to 2016, but looks good anyway :+1:

GianlucaGuarini commented:
@aMarCruz It's in alphabetical order by surname :)

cognitom commented on LICENSE.txt:
```diff
@@ -1,6 +1,13 @@
-Copyright (c) 2015 Muut Inc. + contributors
+Copyright (c) 2015-present
+Originally created by Muut Inc.
+Actively mantained by:
```
ma[i]ntained?

rsbondi replied:
correct @cognitom

GianlucaGuarini replied:
thanks @cognitom I will fix it asap

tipiirai replied:
looks good

GianlucaGuarini commented:
I have fixed the typo and added the donation campaign badge. We can check at anytime the amount of donations to riot and then Tero will be happy to share them with us :v:
